README.md

```diff
@@ -6,6 +6,139 @@
 
 Part of the [OpenTOSCA Ecosystem](http://www.opentosca.org)
 
+## User Guide
+
+### Test Environment Connection
+```java
+public class ContainerClientExamples {  
+      public void testClientConnection() {
+          String dockerHost = "localhost";
+          Integer port = 1337;
+          ContainerClient containerClient = new ContainerClientBuilder()
+                  .withHostname(dockerHost)
+                  .withPort(port)
+                  .build();
+          containerClient.getApplications();
+      }
+  }
+```
+
+### Upload an Application
+```java
+public class ContainerClientExamples {
+        public void uploadApplication() {
+            String dockerHost = "localhost";
+            Integer port = 1337;
+            ContainerClient containerClient = new ContainerClientBuilder()
```

Comment:
Once we showed how to create a client, this is no longer required. Just show and explain (with text or comments) how to use the actions ... and return types of course.